keyphrase_extractor.py

```diff
@@ -7,10 +7,12 @@
 
 from collections import Counter
 
-nltk.download('punkt',quiet='true')
-nltk.download("averaged_perceptron_tagger",quiet='true')
-nltk.download("wordnet",quiet='true')
-nltk.download("brown",quiet='true')
+nltk.download('punkt')
+nltk.download("averaged_perceptron_tagger")
+nltk.download("wordnet")
+nltk.download("brown")
+nltk.download("stopwords")

```

Comment:
`stopwords` was missing before and the code would fail.
Ideally, these download statements shouldn't be here. We should move them in a setup script running just once. @jasonrizk